test/usecases/leonhard/simulate.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+from neuron import h, gui
+from neuron.units import ms
+
+nseg = 1
+
+s = h.Section()
+s.insert("leonhard")
+s.nseg = nseg
```

Comment:
The `Section` creation (apart from the mechanism insertion code) could be moved in a separate function and file since this is going to be used in multiple test files. Feel free to do this in a separate PR if you prefer